core/src/main/java/org/openedx/core/data/model/Block.kt

```diff
@@ -58,7 +60,8 @@ data class Block(
             studentViewData = studentViewData?.mapToDomain(),
             studentViewMultiDevice = studentViewMultiDevice ?: false,
             blockCounts = blockCounts?.mapToDomain()!!,
-            completion = completion ?: 0.0
+            completion = completion ?: 0.0,
+            containsGatedContent = containsGatedContent
```

Comment:
Can we set the default type to `false` and make it non-null in the domain and db layer?  Because we are already considering it false in the domain layer.

Reply:
done

Reply:
This will still be nullable in the `data/model/Block.kt` here: 
```
    @SerializedName("contains_gated_content")
    val containsGatedContent: Boolean?,
```

And we will add a default value in the `mapToDomain` function only.
```
    containsGatedContent = containsGatedContent ?: false

Reply:
I changed it for data.model.Block